packages/util-dynamodb/src/convertToAttr.ts

```diff
@@ -0,0 +1,117 @@
+import { AttributeValue } from "@aws-sdk/client-dynamodb";
+
+import { NativeAttributeValue } from "./models";
+
+/**
+ * Convert a JavaScript value to its equivalent DynamoDB AttributeValue type
+ *
+ * @param {NativeAttributeValue} data The data to convert to a DynamoDB AttributeValue
+ * @param {Object} options
+ * @param {boolean} options.convertEmptyValues Whether to automatically
+ *                      convert empty strings, blobs, and sets to `null`
+ */
+export const convertToAttr = (
+  data: NativeAttributeValue,
+  options?: {
+    convertEmptyValues: boolean;
```

Comment:
just a note that this config is a bit confusing in the current document client, and might be better handled with custom premarshall https://github.com/aws/aws-sdk-js-v3/issues/1533

It's confusing because `""` is considered an empty value, but `undefined` is not

Reply:
The parameter `convertEmptyValues` is added for backward compatibility with v2 DocumentClient
API Reference: https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/DynamoDB/DocumentClient.html